recipes/m2w64-sysroot/meta.yaml

```diff
@@ -0,0 +1,165 @@
+{% set name = "m2w64-sysroot" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ GIT_DESCRIBE_TAG }}.r{{ GIT_DESCRIBE_NUMBER }}.g{{ GIT_DESCRIBE_HASH }}
+
+source:
+  git_url: https://github.com/mingw-w64/mingw-w64
+  git_rev: dc42231f0
```

Comment:
I know it is annoying but I prefer the url archive format and checksum, that brings the format it closer most of our recipes.